### PR TITLE
Add a method to cancel the previous running builds of a given job

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:** 

Adds a method that runs as the first step in the pipeline to check to see if any previous iterations of the same job are building and issues a stop command to cancel those jobs.

This should limit the amount of concurrent running Jenkins jobs the simorgh teams has, especially as push often.

This only runs on PR's and will not stop a build of latest.

Builds with a grey icon have been stoped by the next running build.
image

### CODE REVIEW

**Review by DemonusPC — APPROVED** [review]
Dank

File: Jenkinsfile [hunk 1]
```diff
@@ -80,6 +80,21 @@ def buildStaticAssets(env, tag) {
   stash name: "staticAssets${tag}", includes: "static${tag}.zip"
 }
 
+def cancelPreviousBuilds() {
+  def jobName = env.JOB_NAME
```
Comment (jamesdonoh) [inline Jenkinsfile:84]: jobName here is what, the branch name?

**Review by jamesdonoh — APPROVED** [review]
Looks reasonable as far as I can tell!

File: Jenkinsfile (hunk 1 above)
Comment (chris-hinds) [reply Jenkinsfile:84]: ` Name of the project of this build. This is the name you gave your job when you first set it up` So as this is a multibranch pipeline setup our build names follow the structure `pipeline_name/branch_name` e.g. `simorgh/cancel-previous-builds`

**Pull request merged**